registry/storage/linkedblobstore.go

```diff
@@ -231,7 +231,47 @@ func (lbs *linkedBlobStore) Delete(ctx context.Context, dgst digest.Digest) erro
 	return nil
 }
 
-func (lbs *linkedBlobStore) Enumerate(ctx context.Context, ingestor func(digest.Digest) error) error {
+// LayersEnumerate find layer's digest in the Repository.
+func LayersEnumerate(ctx context.Context, storageDriver driver.StorageDriver, repoName string, ingester func(digest.Digest, string) error) error {
```

Comment:
@dmage Thank you for your review.
I deleted LayerEnumerate(), and use linkedBlogStore.Enumerate.